sklearn/linear_model/coordinate_descent.py

```diff
@@ -1420,12 +1418,13 @@ class ElasticNetCV(LinearModelCV, RegressorMixin):
         Length of the path. ``eps=1e-3`` means that
         ``alpha_min / alpha_max = 1e-3``.
 
-    n_alphas : int, optional
+    n_alphas : int, optional (default=10)
```

Comment:
It seems kinda odd that here some parameters mention their default in the docstring, some don't.

Reply:
Yes thanks, it's also unrelated to this PR so I reverted the changes.

Reply:
All parameters should mention defaults at all times. But it's unrelated, so I don't mind.